services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -297,9 +277,9 @@ getAllFeatures ::
     r =>
   UserId ->
   TeamId ->
-  Sem r Aeson.Value
+  Sem r Aeson.Object
 getAllFeatures uid tid = do
-  Aeson.object
+  KeyMap.fromList
```

Comment:
Why is there a need for this change?

Reply:
because the old endpoint returned just some JSON and it didn't matter if this was an object or a value, but now we return `AllFeatureConfigs` which is defined like this:

```
newtype AllFeatureConfigs = AllFeatureConfigs {_allFeatureConfigs :: Aeson.Object}
```